src/index.ts

```diff
@@ -1,4 +1,4 @@
-import https, { RequestOptions } from 'https';
+import { RequestOptions } from 'https';
```

Comment:
Sweep: Use the `import type` declaration to be specific you are importing a type.